indico/modules/events/layout/templates/page.html

```diff
@@ -3,6 +3,15 @@
 {% block title %}
     {# The title of a user-generated page isn't translated, localized title is not needed #}
     {{ page.menu_entry.title }}
+    {% if event.can_manage(session.user) %}
+    <a href="#"
+        class="icon-edit"
+        data-href="{{ url_for('event_layout.menu_entry_edit', page.menu_entry) }}"
```

Comment:
I think you can omit `event_layout` here

Reply:
When called only from within the same blueprint you can use the relative version. However, this is not the case here since you can mark a custom page as the default event homepage and in that case it will be rendered from a different blueprint.